### DISCUSSION TITLE
Italic comments seem to be rendering illegibily

### DISCUSSION THREAD

**Discussion by hazel-ocean** [helix-editor/helix#7412] [Troubleshooting] [answered]
### Summary

It looks like a number of themes display commands as `italic`; which renders quite poorly on my machine. I've tried this out in Wezterm, and iTerm which displays this behavior (however Alacritty is fine). I think it's probably related to how fonts are being rendered; but I am not sure.

Does someone know more about why this might be?

Here's some screens comparing italics vs non italics

## Wezterm Italics
<img width="1512" alt="Screenshot 2023-06-20 at 5 55 47 PM" src="https://github.com/helix-editor/helix/assets/6754950/f71e7648-1aa0-42de-83f6-a047b7ce1cd3">

## Wezterm Non-Italics
<img width="715" alt="Screenshot 2023-06-20 at 6 01 17 PM" src="https://github.com/helix-editor/helix/assets/6754950/89f533de-0832-4e84-b681-e302cc60d240">

## Alacritty Italics
<img width="616" alt="Screenshot 2023-06-20 at 5 58 49 PM" src="https://github.com/helix-editor/helix/assets/6754950/f8832925-01cd-4cab-93a2-fb38b0c1e3ca">

## Alacritty Non-Italics
<img width="651" alt="Screenshot 2023-06-20 at 6 01 13 PM" src="https://github.com/helix-editor/helix/assets/6754950/4e605719-74ec-4899-8e70-1ef72a0a2160">

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

macOS

### Terminal Emulator

wezterm 20230408-112425-69ae8472 ++ alacritty 0.12.0

### Helix Version

helix 23.05

**Comment by the-mikedavis** [comment, marked as answer]
Are you using tmux? See https://github.com/tmux/tmux/wiki/FAQ#i-dont-see-italics-or-italics-and-reverse-are-the-wrong-way-round, https://github.com/helix-editor/helix/issues/3099

**Reply by hazel-ocean** [reply]
I most definitely am!

**Reply by hazel-ocean** [reply]
Wow, that was a rabbit hole, thank you for sharing this! Turns out that simply setting tmux's default terminal to `tmux-256color` was not sufficient to fix this case for me. I also needed an up-to-date `infocmp` in my PATH so that the proper escape codes could be emitted.

Tested with `TERM=tmux-256color infocmp -1 | grep -E 'ritm|sitm'` with the system's default which doesn't recognize `tmux-256color` and once I installed (with Nix, I haven't tried this with Homebrew or Macports) with the latest `ncurses` library installed in my system's environment which DOES recognize `tmux-256color`.

While this does seem like a `tmux` thing I've run into, I'm curious why Alacritty isn't affected by this behavior.

**Reply by hazel-ocean** [reply]
For those looking at this, if you're running on macOS like I am; you can see that your system `infocmp` can't work with `tmux-256color` by running `TERM=tmux-256color /usr/bin/infocmp -1 | grep -E 'ritm|sitm'`

If you run `which infocmp` and get this same path you may have a way out by installing `ncurses` globally like I did (which also brought with it a newer version of `infocmp`)
